tools/test.py

```diff
@@ -285,7 +289,7 @@ def HasRun(self, output):
       (duration.seconds + duration.days * 24 * 3600) * 10**6) / 10**6
 
     logger.info('  ---')
-    logger.info('  duration_ms: %d.%d' % (total_seconds, duration.microseconds / 1000))
+    logger.info('  duration_s: %d.%d' % (total_seconds, duration.microseconds / 1000))
```

Comment:
no, this is wrong, it doesn't mean "duration in milliseconds", it means "duration with milliseconds"